src/Subscriptions/resources/components/SubscriptionsListTable.tsx

```diff
@@ -71,6 +80,56 @@ const bulkActions: Array<BulkActionsConfig> = [
             </>
         ),
     },
+    {
+        label: __('Trash', 'give'),
+        value: 'trash',
+        type: 'warning',
+        isVisible: (data, parameters) => parameters.status !== 'trash',
+        action: async (selected) => {
+            const response = await API.fetchWithArgs('/setStatus', {ids: selected.join(','), status: 'trashed'}, 'POST');
```

Comment:
@alaca trashing is supposed to send a `DELETE` request without the force option.  This should trigger the model `$subscription->trash()` method.  This is a little different than just a status update because it preserves the old status like how `wp_trash_post` and  `wp_untrash_post` works so we can restore the item back to it's original state.